index.js

```diff
@@ -39,73 +39,103 @@ const getPackageJson = function() {
   return pkg;
 };
 
-const defaultGlobals = {
-  browser: {
-    'video.js': 'videojs',
-    'global': 'window',
-    'global/window': 'window',
-    'global/document': 'document'
-  },
-  module: {
-    'video.js': 'videojs'
-  },
-  test: {
-    'qunit': 'QUnit',
-    'qunitjs': 'QUnit',
-    'sinon': 'sinon',
-    'video.js': 'videojs'
-  }
-};
-
-const defaultExternals = {
-  browser: [],
-  module: [
-    'global',
-    'global/document',
-    'global/window'
-  ],
-  test: []
-};
-
-const defaultPlugins = {
-  browser: [
-    'resolve',
-    'json',
-    'commonjs',
-    'uglify',
-    'babel'
-  ],
-
-  module: [
-    'resolve',
-    'json',
-    'commonjs',
-    'babel'
-  ],
+// if checkWatch is true and -w or --watch was passed to rollup
+// we should "change the build" for rollups watch mode.
+const shouldChangeWatch = (settings) =>
+  settings.checkWatch && process.argv.some((arg) => (/^-w|--watch$/).test(arg));
+
+// verify that listed plugins are an object or in primed plugins
+const validatePlugins = ({plugins, primedPlugins}) =>
```

Comment:
Made this a function so its more clear what we are doing and to simplify `getSettings`